README.md

```diff
@@ -1,23 +1,35 @@
 # Talk [![CircleCI](https://circleci.com/gh/coralproject/talk.svg?style=svg)](https://circleci.com/gh/coralproject/talk)
 A commenting platform from The Coral Project. [https://coralproject.net](https://coralproject.net)
 
+## Contributing to Talk
+
+### Local Dependencies
+Node
+Mongo
+
 ### Getting Started
 `npm install`
 Run it once to install the dependencies.
 
 `npm start`
 Runs Talk.
 
+### Running with Docker
+Make sure you have Docker running first and then run `docker-compose up -d`
+
 ### Testing
 `npm test`
 
 ### Lint
 `npm run lint`
 
+### Helpful URLs
+Bare comment stream: http://localhost:5000/client/coral-embed-stream
```

Comment:
Oh yeah if I use docker, it comes out as 5000, but we can just note PORT so they can run on whatever port they're using.